canonical-websites/www.ubuntu.com
# Updated maas contact us and thank you pages to use server default + about page update

## Done

* removed about from navigation.yaml so it doesn't get a secondary nav, per MS
* removed maas specific contact-us and thank-you pages
* moved maas to the server/ directory
* updated the copy docs
* added redirects
* updated links to the old maas contact-us page
* fixed a bad link to jujucharms.com

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- see that [maas](http://0.0.0.0:8001/server/maas) points to the new [contact-us form](http://0.0.0.0:8001/server/contact-us?product=server-maas)
- see that the thank-you form looks correct (you cannot test from filling out the form due to the way marketo works) http://0.0.0.0:8001/server/thank-you?product=server-maas
- see that the [about us page](http://0.0.0.0:8001/about) doesn't have a secondary nav
